test/ibm/runtime/test_runtime.py

```diff
@@ -427,6 +427,27 @@ def test_run_program_with_custom_runtime_image(self):
         self.assertTrue(job.result())
         self.assertEqual(job.image, image)
 
+    def test_retrieve_program_data(self):
+        """Test retrieving program data"""
+        program_id = self._upload_program(name="qiskit-test")
+        self.runtime.programs()
+        program = self.runtime.program(program_id)
+        self.assertEqual(program.data, "def main() {}")
```

Comment:
Can we make a constant DEFAULT_DATA = "def main() {}" on top of this file and use it everywhere in this file.